Plash/AppDelegate.swift

```diff
@@ -247,10 +247,25 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 	func loadURL(_ url: URL?) {
 		webViewError = nil
 
-		guard let url = url else {
+		guard var url = url else {
 			return
 		}
 
+		// Sub out placeholder variables for actual values.
+
+		// Here we swap out [screenWidth] and [screenHeight] for their actual values.
+		// We proceed only if we have an NSScreen to work with.
+		if let screen = desktopWindow.targetScreen?.withFallbackToMain ?? .main {
+			let screenResolution = screen.visibleFrameWithoutStatusBar
+			let urlString = url.absoluteString
+				.replacingOccurrences(of: "%5BscreenWidth%5D", with: String(format: "%.0f", screenResolution.width))
```

Comment:
Can you escape it in code instead of pre-escaping it? For readability.

It might be worth generalizing some of the logic here so it would be easier to add additional variables in the future.

Reply:
Have a look at the changes I made in my latest commit, particularly [`URLPlaceholder`](https://github.com/sindresorhus/Plash/pull/56/commits/4f3c2a4bddf72c410b13afeb19387849c3c1cfe5#diff-04cd44617f340bfa63fd4b9dcd27d7d7R14).